samples/nrf5340/netboot/src/main.c

```diff
@@ -41,6 +44,20 @@ int main(void)
 	bool valid = false;
 	uint8_t status = pcd_fw_copy_status_get();
 
+#ifdef CONFIG_PCD_LOCK_NETCORE_DEBUG
+	if (status == PCD_STATUS_LOCK_DEBUG) {
```

Comment:
Why don't we use a switch statement to check the status? And its not really a status any more right? Its a command at this point.

Reply:
I think that status reflects what we are doing at the moment. At this point we are locking debug. After that, it is done.

That said, `PCD_COMMAND_LOCK_DEBUG` would work fine as a name for this.